packages/common/src/sources/functions/getExternalSourceOrigin.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Soliguide: Useful information for those who need it
+ *
+ * SPDX-FileCopyrightText: © 2024 Solinum
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import { PAIRING_SOURCES } from "../constants";
+import { PairingSources } from "../enums";
+
+export interface ExternalSourceOrigin { name: PairingSources; isOrigin?: boolean }
```

Comment:
```suggestion
export interface ExternalSourceOrigin {
  name: PairingSources; 
  isOrigin?: boolean
}
```

T'es sûr qu'elle existe pas cet interface déjà ? Ou un équivalent où tu pourrais mettre Pick ?